<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>aboutWindow</class>
 <widget class="QDialog" name="aboutWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="helpButton" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>462</y>
     <width>193</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>help</string>
   </property>
  </widget>
  <widget class="QToolButton" name="configButton" >
   <property name="geometry" >
    <rect>
     <x>407</x>
     <y>462</y>
     <width>50</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>config...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="updateCheckBox" >
   <property name="geometry" >
    <rect>
     <x>208</x>
     <y>462</y>
     <width>193</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>update</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>193</width>
     <height>20</height>
    </rect>
   </property>
   <property name="title" >
    <string>GroupBox</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>35</y>
     <width>448</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>8</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QTextBrowser" name="textBrowser" >
      <property name="readOnly" >
       <bool>false</bool>
      </property>
      <property name="html" >
       <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:MS Shell Dlg; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>&lt;span style=&quot; color:#000000;&quot;>This is a Qt demo application for a skin system.&lt;/span>&lt;/p>&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;>&lt;/p>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;>The skin system works as a Qt style plugin through the QStyle class.&lt;/p>&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;>&lt;/p>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;>This example works using Qt version 4.0.0 &lt;/p>&lt;/body>&lt;/html></string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
